src/Avalonia.Diagnostics/Diagnostics/ViewModels/LogicalTreeNode.cs

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) The Avalonia Project. All rights reserved.
```

Comment:
Any reason why half the files have license header and half do not?

Reply:
Simply, the ones where I modified an old file still have headers. I don't bother adding headers to files these days as it just feels like busywork and from what I've read legally it's meaningless.

Reply:
Maybe I should delete existing headers on modified files? Not sure.

Reply:
TBH it would be good to be consistent at least. I have no problem with or without license info as long as we stick to one option.